_data/ols-1-projects.yaml

```diff
@@ -147,7 +147,6 @@
     we can benefit life sciences sector of whole country through open science
     initiative. I also want to make this initiative feasible and sustainable by creating
     incoming generating platform with computational biology.
-  visible: false
```

Comment:
Are they ok to make the project visible? They were not initially

Reply:
Sudarshan mentioned that he is ok with it.